os/arch/arm/src/stm32/stm32_otgfsdev.c

```diff
@@ -1012,10 +1055,10 @@ static void stm32_txfifo_write(FAR struct stm32_ep_s *privep, FAR uint8_t *buf,
 		 * and pack these into one 32-bit word (little endian).
 		 */
 
-		regval = (uint32_t)*buf++;
-		regval |= ((uint32_t)*buf++) << 8;
-		regval |= ((uint32_t)*buf++) << 16;
-		regval |= ((uint32_t)*buf++) << 24;
+		regval = (uint32_t) * buf++;
+		regval |= ((uint32_t) * buf++) << 8;
+		regval |= ((uint32_t) * buf++) << 16;
+		regval |= ((uint32_t) * buf++) << 24;
```

Comment:
please revert these changes. This is abnormal change of formatter. Previous codes are right.

Reply:
done